Pod/Classes/PagingMenuController.swift

```diff
@@ -321,8 +323,14 @@ extension PagingMenuController: UIScrollViewDelegate {
         
         move(toPage: nextPage)
     }
-    
+
+    public func scrollViewWillBeginDragging(_ scrollView: UIScrollView) {
```

Comment:
`scrollViewWillBeginDecelerating(_:)` is appropriate for `. didScrollStart` since we can scroll by calling `move(toPage:)` without dragging.